src/components/atoms/formInput/InpiutInterface.ts

```diff
@@ -0,0 +1,9 @@
+import { RegisterOptions } from "react-hook-form";
```

Comment:
spelling mistake in file name? also a file that only has interface/types should be named as type.ts